docs/user-installation-checklist.md

```diff
@@ -0,0 +1,77 @@
+# OpenShift Provider Certification Tool - Cluster Installation Checklist
+
+<!--
+Do not change the following markdown commented lines.
+__version__: 0.1.0-preview
+-->
+
+This checklist should be provided for every new support case, or when any items have been changed (for Example, Compute flavor). This will help the Red Hat engineers while working in your partner support case during the certification process.
+
+If you have any questions you can:
+
+- Review the [Installation Review Guide](./user-installation-review.md)
+- Review the [OpenShift Infrastructure Provider Guide](https://docs.providers.openshift.org/)
+- Review the [OpenShift Documentation Page related to the version your are certifying](https://docs.openshift.com/container-platform)
+- Ask your Red Hat Partner
+
+## Provider Information
+
+- Provider Name:                # Example: MyCloud
+- OpenShift Version:            # Example: 4.11.4
+- Certification Type:           # Options: (Standard | Upgrade)
+- Environment Setup Topology:   # Options: (Standard | Dedicated)
+
+## Review Checklist
+
+### Compute
+
+- [ ] The Control Plane nodes meet the minimum requirements
+- [ ] The Compute nodes meet the minimum requirements
```

Comment:
Unfortunately, the flavor of markdown used by mkdocs is different than GitHub. These checkboxes do not render with `mkdocs serve` :( 

What do you think we should do here?

Reply:
Looks like there cannot be a space between the brackets for mkdocs to render the checkbox

Reply:
https://github.com/mkdocs/mkdocs/issues/1660

Reply:
I think we should keep this as the focus is on the partner checking it and submitting the file.

mkdocs is a plus =)